src/components/transactions/Supply/SupplyModalContent.tsx

```diff
@@ -212,7 +213,11 @@ export const SupplyModalContent = ({
       {showIsolationWarning && <IsolationModeWarning asset={poolReserve.symbol} />}
       {supplyCap.determineWarningDisplay({ supplyCap })}
       {debtCeiling.determineWarningDisplay({ debtCeiling })}
-      {poolReserve.symbol === 'AMPL' && <AMPLWarning />}
+      {poolReserve.symbol === 'AMPL' && (
+        <Warning sx={{ mt: '16px', mb: '40px' }} severity="warning">
+          <AMPLWarning />
+        </Warning>
+      )}
```

Comment:
I suppose these types are checks are okay in the interim until we figure out a better approach